homeassistant/config_entries.py

```diff
@@ -705,6 +705,11 @@ def async_update_entry(self, entry, *, data=_UNDEF, options=_UNDEF):
         except loader.IntegrationNotFound:
             raise data_entry_flow.UnknownHandler
 
+        try:
+            integration.get_component()
+        except ImportError:
```

Comment:
We should probably print an error here and when an integration is not found (block above this one), so that a user can know why an integration doesn't work.